pkg/kubelet/dockertools/manager.go

```diff
@@ -54,8 +54,17 @@ const (
 
 	maxReasonCacheEntries = 200
 
-	kubernetesPodLabel       = "io.kubernetes.pod.data"
-	kubernetesContainerLabel = "io.kubernetes.container.name"
+	// In order to avoid unnecessary SIGKILLs, give every container a minimum grace
```

Comment:
This is really just to guard against *forcing* SIGKILL.  We default pod.Spec.TerminationGracePeriodSceonds to 30 - this just guarantees you never get less than 2s to respond to SIGTERM.